Fix (#3)
sendbird/healthcare-ai-chatbot: README.md

> This demo app demonstrates the implementation of the AI Chatbot tailored for healthcare. It includes functionalities such as ***recommend date to reservation, reservation, Talking to the doctor.*** By leveraging ChatGPT’s new feature [Function Calling](https://openai.com/blog/function-calling-and-other-api-updates), the Chatbot now can make an API request to the 3rd party with a predefined Function Calling based on customer’s enquiry. Then it parses and presents the response in a conversational manner, enhancing overall customer experience.

Reviewer: ```suggestion
This demo app demonstrates the implementation of the AI Chatbot tailored for healthcare. It includes functionalities such as ***suggesting possible appointment dates, making an appointment, and talking to the doctor.*** By leveraging ChatGPT’s new feature [Function Calling](https://openai.com/blog/function-calling-and-other-api-updates), the Chatbot now can make an API request to the 3rd party with a predefined Function Calling based on customer’s enquiry. Then it parses and presents the response in a conversational manner, enhancing overall customer experience.
```

                    link: nil
                )

                return [orderParams]
            }
            if let items = try?SBUGlobalCustomParams.cardViewParamsCollectionBuilder?(response.rawString()!){
                self.addCardListView(with: items)
            }
        } else if endpoint.contains("/dates/recommended") {
            // Replace the message text with the custom text
            customText = "Here are the available appointment date and time."
            disableWebview = true
            SBUGlobalCustomParams.cardViewParamsCollectionBuilder = { messageData in
                guard
                    let data = messageData.data(using: .utf8),
                    let json = try? JSON(data: data)
                else { return [] }
                return json.arrayValue.compactMap { item in
                    return SBUCardViewParams(
                            imageURL: nil,
                            title: "\(item["doctor"].stringValue)",
                            subtitle: "Date: \(item["recommend_date"].stringValue)",
                            description: nil,
                            link: nil,
                            actionHandler: {

>                                 self.cardSelectHandler!("Please reserve a reservation with \(item["doctor"].stringValue), \(item["recommend_date"].stringValue)")

Reviewer: ```suggestion
                                self.cardSelectHandler!("Please make an appointment with \(item["doctor"].stringValue), \(item["recommend_date"].stringValue)")
```
